QUICKSTART.rst

```diff
@@ -3,120 +3,150 @@
 Chapel Quickstart Instructions
 ==============================
 
-The following instructions are designed to help you build Chapel from
-source.  If you haven't already downloaded Chapel, obtain the latest
-release from http://chapel.cray.com/download.html, or refer to
-http://chapel.cray.com/install.html for other installation options.
+These instructions are designed to help you get started with
+a source distribution of Chapel.
 
 In the following instructions, note that building and using Chapel as
-described in steps 1-7 disables optional and advanced features in the
-interest of getting you a clean build as quickly as possible.  Step 8
-explains how to re-build in the preferred configuration with these
-additional features enabled.  Step 9 explains how to build Chapel for
-distributed memory execution.
+described in steps 2-5 disables optional and advanced features in the
+interest of getting you a clean build as quickly as possible. Later 
+sections explain how to re-build in the preferred configuration and how to
+enable more features, such as distributed memory execution.
 
 
 0) See `doc/prereqs.rst`_ for more information about system tools and
    packages you may need to have installed to build and run Chapel.
 
-.. _doc/prereqs.rst: http://chapel.cray.com/docs/1.13/usingchapel/prereqs.html
+1) If you don't already have Chapel 1.14, see
```

Comment:
Maybe specify - "If don't already have Chapel 1.14 downloaded, see .."

I was going to suggest dropping the 1.14, but I guess we'll be updating version number here anyway.